src/components/unordered-list/unordered-list.scss

```diff
@@ -12,7 +12,7 @@ $list: null !default;
 .ecl-unordered-list,
 %ecl-unordered-list {
   color: map.get($list, 'text-color');
-  font: var(--f-m);
+  font: map.get($list, 'unordered', 'font');
```

Comment:
this was not really needed, when we don't have differences in the two systems there is no need to parametrize values. Fine, anyway